sqlglot/parser.py

```diff
@@ -108,6 +108,8 @@ class Parser(metaclass=_Parser):
         TokenType.CURRENT_USER: exp.CurrentUser,
     }
 
+    JOIN_HINTS = set()  # type: set[TokenType]
```

Comment:
Note: this should be `t.Set[str]` if we don't use tokens for the join hints.

```suggestion
    JOIN_HINTS: t.Set[TokenType] = set()
```

Reply:
Will change that indeed.